cluster-scope/base/core/namespaces/octo-et-security-demo-next/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+components:
+- ../../../../components/project-admin-rolebindings/octo-et-security
+- ../../../../components/limitranges/default
+- ../../../../components/resourcequotas/medium
```

Comment:
The resource quota for the onboarding issue is stated as `small` however this one is medium. It does say in the issue thread that this conversation took place over google chat, can I confirm that this medium sizing is required?

Reply:
I already bumped it to medium, since small is probably too small 🤷